### PR TITLE
renamed enter-side

### PR DESCRIPTION
**Pull request by SmartManoj**
Thank you for sending your pull request. But first, have you included
unit tests, and is your code PEP8 conformant? [More details](https://github.com/freqtrade/freqtrade/blob/develop/CONTRIBUTING.md)

## Summary

renamed enter-side

### CODE REVIEW

**Comment by SmartManoj** [comment]
For enter_long and enter_short, long_entry and short_entry would be appropriate. Suggestions?

File: freqtrade/persistence/models.py [hunk 1]
```diff
@@ -371,7 +371,7 @@ def close_date_utc(self):
         return self.close_date.replace(tzinfo=timezone.utc)
 
     @property
-    def enter_side(self) -> str:
+    def entry_side(self) -> str:
```
Comment (xmatthias) [inline freqtrade/persistence/models.py:380]: Please add a (temporary) compatibility layer here (for strategies using this already with develop).

This means - a property named `enter_side` (which calls `entry_side`). 
It should have the docstring of "DEPRECATED, please use entry_side` - and a TODO with "remove me after 2022.5".

**Review by xmatthias — COMMENTED** [review]
> For enter_long and enter_short, long_entry and short_entry would be appropriate. Suggestions?

I don't see any benefit from this renamal - therefore please leave that as is.
We've been working on feat/short for almost a year - and this part has been changed roughly half a year ago - enough time for anyone to step in and suggest a naming change.

We'll not change it again at this point.

File: freqtrade/persistence/models.py (hunk 1 above)
Comment (xmatthias) [reply freqtrade/persistence/models.py:380]: I'm impatient - fixed it myself ... :laughing:

**Review by xmatthias — APPROVED** [review]
Nice job :+1: 

Thanks a lot for your contribution! :100:

**Pull request merged**